games-roguelike/hengband/files/hengband-1.6.2-autoconf-ncurses.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/configure.in b/configure.in
+index 0df0e76..91649ee 100644
+--- a/configure.in
++++ b/configure.in
+@@ -56,13 +56,7 @@ fi
+ 
+ dnl Checks for libraries.
+ dnl Replace `main' with a function in -lncurses:
+-AC_CHECK_LIB(ncurses, initscr, [AC_DEFINE(USE_GCU, 1, [Allow -mGCU environment]) AC_DEFINE(USE_NCURSES, 1, [Use ncurses]) LIBS="$LIBS -lncurses"])
+-if test "$ac_cv_lib_ncurses_initscr" != yes; then
+-  AC_CHECK_LIB(curses, initscr, [AC_DEFINE(USE_GCU, 1, [Allow -mGCU environment]) LIBS="$LIBS -lcurses"])
+-  if test "$ac_cv_lib_curses_initscr" != yes; then
+-    AC_CHECK_LIB(termcap, tgetent, [AC_DEFINE(USE_CAP, 1, [Allow -mCAP environment]) LIBS="$LIBS -ltermcap"])
+-  fi
+-fi
++PKG_CHECK_MODULES([NCURSES], [ncurses], LIBS="${LIBS} ${NCURSES_LIBS}")
```

Comment:
`PKG_CHECK_MODULES([NCURSES], [ncurses], [LIBS="${LIBS} ${NCURSES_LIBS}"])`

Reply:
Nice catch @SoapGentoo I'll fix this